integration/spark-common-test/src/test/scala/org/apache/carbondata/spark/testsuite/standardpartition/StandardPartitionWithPreaggregateTestCase.scala

```diff
@@ -225,6 +227,26 @@ class StandardPartitionWithPreaggregateTestCase extends QueryTest with BeforeAnd
     checkAnswer(sql("select * from partitionone_p1"), Seq(Row("k",2014,2014,1,2), Row("k",2015,2015,2,3)))
   }
 
+  test("test drop partition directory") {
+    sql("drop table if exists droppartition")
+    sql(
+      """
+        | CREATE TABLE if not exists droppartition (empname String)
```

Comment:
forget to drop this table after test?

Reply:
if you check the brefore  and after all, it is complete new database, so no need to do it